sdk/ai/azure-ai-projects/samples/evaluations/sample_evaluations.py

```diff
@@ -54,26 +53,28 @@
     data=Dataset(id=data_id),
     evaluators={
         "f1_score": EvaluatorConfiguration(
-            id=F1ScoreEvaluator.evaluator_id,
+            # id=F1ScoreEvaluator.id,
+            id="azureml://registries/azureml-staging/models/F1Score-Evaluator/versions/3",
         ),
         "relevance": EvaluatorConfiguration(
-            id=RelevanceEvaluator.evaluator_id,
+            # id=RelevanceEvaluator.id,
+            id="azureml://registries/azureml-staging/models/Relevance-Evaluator/versions/3",
```

Comment:
Is the line ` # id=RelevanceEvaluator.id,` needed?

Reply:
Yes that will be the final sample. However I am waiting for evaluators to get published and azure-ai-evaluation package to reflect it.